modules/hal_silabs/simplicity_sdk/Kconfig

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2025 Silicon Laboratories Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+menu "SiSDK configuration"
+	depends on HAS_SILABS_SISDK
+
+config RAIL_PA_CURVE_HEADER
+	string "RAIL PA custom curve header file"
+	default "pa_curves_efr32.h"
+	help
+	  Name of custom PA curve header file for use by PA initialization.
+	  See AN1127 for information on how to create a custom PA curve.
```

Comment:
To adhere to the Zephyr standards, this Kconfig option could be overloaded in every `zephyr/boards/silabs/*/*_defconfig` (or in `soc/silabs/*/Kconfig.defconfig`). So the final PA curves is exposed to the user. That said, this can be done later.